Automattic/woocommerce-payments
# Migrate components chip and tests to TypeScript

Example component migration. Depends on #1263.

The `<Chip />` component is an easy one to migrate because it has no dependencies, so I thought I'd use it to demonstrate how to migrate our components to TypeScript.

#### Changes proposed in this Pull Request

- The `<Chip />` component is migrated to TypeScript

#### Testing instructions





* `npm i`
* `npm run watch`
* Open any UI in WCPay that uses the `<Chip />` component, e.g. any of the details screens.
* Make sure the chips render correctly

* `npm i`
* `npm run test:js -- client/components/chip`
* Make sure tests pass

-------------------
